.github/workflows/pull_request.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           pylint UnleashClient
       - name: Unit tests
         run: |
-          py.test --html=pytest/results.html --junitxml=pytest/results.xml --self-contained-html --cov=UnleashClient tests/unit_tests
+          py.test --flake8 --html=pytest/results.html --junitxml=pytest/results.xml --self-contained-html --cov=UnleashClient tests/unit_tests
```

Comment:
why did you add flake8?

Reply:
`-flake8` was previously only applied to the tests via `pytest.ini`.  The argument causes some headaches with VSCode test descovery, so in the interests of being more dev-agnostic I was moving it to Make/CI until I can figure out how to fix the configs.  

(Longer term I'd like to unify/modernize linting a bit.)

Reply:
`-flake8` was previously only applied to the tests via `pytest.ini`. The argument causes some headaches with VSCode test discovery, so in the interests of being more dev-agnostic I was moving it to Make/CI until I can figure out how to fix the configs.

(Longer term I'd like to unify/modernize linting a bit.)